sqlstore/src/test/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateInstrumentDaoTest.java

```diff
@@ -53,26 +51,26 @@ public void setup() {
 
   @Test
   public void testListAll() throws IOException {
-    Collection<SequencerReference> sequencers = dao.listAll();
-    assertEquals(sequencers.iterator().next().getPlatform().getPlatformType(), PlatformType.ILLUMINA);
+    Collection<Instrument> instruments = dao.listAll();
+    assertEquals(instruments.iterator().next().getPlatform().getPlatformType(), PlatformType.ILLUMINA);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Swap these 2 arguments so they are in the correct order: expected value, actual value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3415)